src/status_im2/contexts/chat/lightbox/zoomable_image/style.cljs

```diff
@@ -22,14 +22,13 @@
     :height          (if set-full-height? "100%" height)}))
 
 (defn image
-  [{:keys [image-width image-height]}
-   {:keys [rotate rotate-scale]}
-   {:keys [border-value images-opacity]}
+  [{:keys [rotate rotate-scale]}
+   {:keys [border-value images-opacity screen-width screen-height]}
    index]
   (reanimated/apply-animations-to-style
    {:transform     [{:rotate rotate}
                     {:scale rotate-scale}]
     :opacity       (nth images-opacity index)
     :border-radius border-value}
-   {:width  image-width
-    :height image-height}))
+   {:width  screen-width
+    :height screen-height}))
```

Comment:
This can't be like that. It has to use `image-width` and `image-height`. Notice how the margins and border radius now disappear